src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/default-transcripts-list-item/DefaultTranscriptListItem.scss

```diff
@@ -12,3 +12,13 @@
   color: $blue;
   cursor: pointer;
 }
+
+.defaultTranscriptLabel {
+  display: flex;
+  justify-content: flex-end;
+  align-items: baseline;
+}
+
+.helpText {
+  max-width: 300px;
```

Comment:
I need to change this. But need help to override the help text css

Reply:
this is no more required.

Reply:
Delete it then? :wink: